com.avaloq.tools.ddk.xtext.generator/src/com/avaloq/tools/ddk/xtext/generator/parser/antlr/AnnotationAwareXtextAntlrGeneratorFragment2.xtend

```diff
@@ -289,10 +292,33 @@ class AnnotationAwareXtextAntlrGeneratorFragment2 extends XtextAntlrGeneratorFra
     ideBindings.contributeTo(language.ideGenModule)
   }
 
-  override protected void addUiBindingsAndImports() {
+  override protected addUiBindingsAndImports() {
     /* Overridden to prevent conflicting bindings related to Content Assist. */

```

Comment:
Could you adapt the comment like this?
/* Overridden to remove the binding for IContentAssistParser, which we bind at a different place, if we would register it here, we would have conflicting bindings for IContentAssistParser. */

Reply:
Done. Thanks for the comment.